proposals/lip-0063.md

```diff
@@ -218,50 +218,72 @@ ffffff
+[LIP 0055](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0055.md#notation-and-constants)               | `CHAIN_ID`                          | `0x 00 00 00 00`
+[LIP 0061](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0061.md#constants)                            | `BLOCK_TIME`                        | 10 seconds
+[LIP 0068](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0068.md#constants)                            | `MAX_PARAMS_SIZE`                   | 14 KiB (14*1024 bytes)
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `FACTOR_SELF_VOTES`                 | 10
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `BASE_VOTE_AMOUNT`                  | `10 * (10)^8`
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `MAX_LENGTH_NAME`                   | 20
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `MAX_NUMBER_SENT_VOTES`             | 10
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `MAX_NUMBER_PENDING_UNLOCKS`        | 20
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `FAIL_SAFE_MISSED_BLOCKS`           | 50
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `FAIL_SAFE_INACTIVE_WINDOW`         | 130,000
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `LOCKING_PERIOD_VOTES`              | 26,000
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `LOCKING_PERIOD_SELF_VOTES`         | 260,000
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `PUNISHMENT_WINDOW_VOTES`           | 260,000
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `PUNISHMENT_WINDOW_SELF_VOTES`      | 780,000
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `POM_LIMIT_BANNED`                  | 5
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `ROUND_LENGTH`                      | 103
+[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `BFT_THRESHOLD`                     | 68
```

Comment:
This constant is not used any more in the DPoS module.

Reply:
Right, thanks. But is this threshold constant now defined in another LIP? I guess this threshold must be set somewhere.

Reply:
It is instead computed in the DPoS module, see the computation before the call of `setValidatorParams`. The comment can be resolved from my side.